README.md

```diff
@@ -2,14 +2,19 @@
 
```

Comment:
The changes to the README should not be in this branch. Can you please revert these?

Reply:
Reverted

Reply:
Perfect, thanks!